0035-ilp-over-http/0035-ilp-over-http.md

```diff
@@ -52,7 +62,60 @@ Content-Type: application/octet-stream
 < Body: Binary OER-Encoded ILP Fulfill or Reject Packet >
 ```
 
-All ILP Packets MUST be returned with the HTTP status code `200: OK`.
+If the request included a `Callback-Url` header and `Request-Id` header, the recipient handling the ILP Prepare SHOULD choose to handle the packet asynchronously and return the corresponding ILP Fulfill/Reject in a separate outgoing HTTP request.
+
+In the asynchronous mode, if the request is semantically valid, the recipient MUST respond immediately that the ILP Prepare is accepted for processing, even if the packet will ultimately be rejected:
+
+```http
+HTTP/x.x 202 Accepted
+```
+
+### Async ILP Fulfill/Reject Reply
+
+#### Request
+
+```http
+POST /ilp HTTP/x.x
+Host: example.com
```

Comment:
Can we change these to reflect the `Callback-Url` used above for consistency.

Reply:
It may also make sense to change the URL of the request and the reply to different domains so it's more obvious they are different parties